examples/webgl_geometry_displacement.html

```diff
@@ -0,0 +1,291 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - geometry - displacement</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				color: #808080;
+				font-family: Monospace;
+				font-size: 13px;
+				text-align: center;
+
+				background-color: #fff;
+				margin: 0px;
+				overflow: hidden;
+			}
+
+			#info {
+				position: absolute;
+				top: 0px; width: 100%;
+				padding: 5px;
+			}
+
+			a {
+
+				color: #0080ff;
+			}
+
+		</style>
+	</head>
+	<body>
+
+		<div id="container"></div>
+		<div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - cpu displacement mapping</div>
+
+		<script src="../build/three.js"></script>
+
+		<script src="js/loaders/TGALoader.js"></script>
+		<script src="js/Detector.js"></script>
+		<script src="js/libs/stats.min.js"></script>
+
+		<script>
+
+			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+			var container, stats;
+
+			var camera, scene, light, pipe, renderer;
+
+			var mouseX = 0, mouseY = 0;
+
+			var time = 0;
+
+			var windowHalfX = window.innerWidth / 2;
+			var windowHalfY = window.innerHeight / 2;
+
+			var diffusePath = "textures/pipe/diffuse.tga";
+			var emissivePath = "textures/pipe/emissive.tga";
+			var heightPath = "textures/pipe/height.tga";
+			var normalPath = "textures/pipe/normal.tga";
+			var specularPath = "textures/pipe/specular.tga";
+
+			init();
+			animate();
+
+			function init() {
+
+				container = document.getElementById( 'container' );
+
+				camera = new THREE.PerspectiveCamera( 20, window.innerWidth / window.innerHeight, 1, 10000 );
+				camera.position.z = 100;
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0x000000 );
+
+				light = new THREE.AmbientLight( 0xffffff );
+				scene.add ( light );
+
+				var displacementLoader = new THREE.TGALoader();
+				displacementLoader.load( heightPath, displacementLoadEvent );
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				container.appendChild( renderer.domElement );
+
+				stats = new Stats();
+				container.appendChild( stats.dom );
+
+				document.addEventListener( 'mousemove', onDocumentMouseMove, false );
+				window.addEventListener( 'resize', onWindowResize, false );
+
+			}
+
+			function displacementLoadEvent( textureData ) {
+
+				var geometry = new THREE.BoxGeometry( 50, 50, 50, 50, 50, 50 );
+
+				applyDisplacementMap( geometry, textureData, 6, 0 );
+
+				var material = new THREE.MeshPhongMaterial();
+				pipe = new THREE.Mesh( geometry, material );
+				scene.add(pipe);
+
+				// map diffuse texture
+
+				var diffuseLoader = new THREE.TGALoader();
```

Comment:
It should be sufficient to create a single instance of `TGALoader`  and reuse it.